i18n/nl_BE.csv

```diff
@@ -158,12 +158,12 @@
 "Magento Enterprise Edition version","Magento Enterprise Edition versie"
 "Quick reference guide","Eerste stappen"
 "Log in to the ","Log in bij "
-"Buckaroo Payment Plaza","Buckaroo Payment Plaza"
+"Buckaroo Buckaroo Plaza","Buckaroo Buckaroo Plaza"
 " to collect the following security settings:"," om de volgende beveiligingsgegevens te verzamelen:"
 "Support","Support"
 "This extension is developed by Buckaroo.","Deze extensie is ontwikkeld door Buckaroo."
 "Extension basic configuration and account information","Algemene instellingen en accountgegevens"
-"Buckaroo Payment Plaza:","Buckaroo Payment Plaza:"
+"Buckaroo Buckaroo Plaza:","Buckaroo Buckaroo Plaza:"
```

Comment:
same

Reply:
**This should be:**
"Buckaroo Plaza:","Buckaroo Plaza:"